### PR TITLE
Datepicker position inside Fullscreen for IE11

### PR DESCRIPTION
**Pull request by hkmarques**

### CODE REVIEW

File: components/fields/forked-react-dates/src/components/SingleDatePicker.js [hunk 1]
```diff
@@ -307,7 +307,7 @@ export default class SingleDatePicker extends React.Component {
             height,
         };
 
-        const hasFullscreenParent = !!element.closest('.m-fullscreen');
+        const hasFullscreenParent = !!$(element).closest('.m-fullscreen');
```
Comment (jimmed) [inline components/fields/forked-react-dates/src/components/SingleDatePicker.js]: WUT

Comment (jimmed) [reply components/fields/forked-react-dates/src/components/SingleDatePicker.js]: NO

Comment (jimmed) [reply components/fields/forked-react-dates/src/components/SingleDatePicker.js]: I didn't fight in 2 browser wars so my kids could use jQuery in React

Comment (jimmed) [reply components/fields/forked-react-dates/src/components/SingleDatePicker.js]: On a more serious note:

```js
const hasFullscreenParent =
  [...document.querySelectorAll('.m-fullscreen')]
    .some(node => node.contains(elements))
```

Comment (hkmarques) [reply components/fields/forked-react-dates/src/components/SingleDatePicker.js]: I'm offensive and I find this jQuery

**Pull request merged**